Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Actually, I mistook it for the (previous) cop which warned people to freeze string constants.